### PR TITLE
KOGITO-2360 - Enable webhooks and comment trigger on Kogito backend P…

### PR DESCRIPTION
**Pull request by MarianMacik**
…R checks

PR set:
* kiegroup/kogito-runtimes#550 
* kiegroup/kogito-apps#258
* kiegroup/kogito-examples#272

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

File: Jenkinsfile [hunk 1]
```diff
@@ -1,5 +1,9 @@
 @Library('jenkins-pipeline-shared-libraries')_
 
+def CHANGE_AUTHOR = "$ghprbPullAuthorLogin"
+def CHANGE_BRANCH = "$ghprbSourceBranch"
+def CHANGE_TARGET = "$ghprbTargetBranch"
+
```
Comment (Ginxo) [inline Jenkinsfile:6]: I would replace them by
```
def CHANGE_AUTHOR = env.CHANGE_AUTHOR ?: ghprbPullAuthorLogin
def CHANGE_BRANCH = env.CHANGE_BRANCH ?: ghprbSourceBranch
def CHANGE_TARGET = env.CHANGE_TARGET ?: ghprbTargetBranch
```
in order to allow either multibranch pipeline job types and regular job types + hooks, wdyt?

Comment (MarianMacik) [reply Jenkinsfile:6]: I can, but I don't plan to go back to multibranch pipeline. But maybe as a safety measure...

Comment (MarianMacik) [reply Jenkinsfile:6]: Fixed. Just prioritized regular job type variables over multibranch ones.

**Pull request merged**